workos/directory_sync.py

```diff
@@ -7,6 +7,7 @@
 from workos.utils.validation import DIRECTORY_SYNC_MODULE, validate_settings
 
 RESPONSE_LIMIT = 10
+ORDER_DEFAULT = "normal"
```

Comment:
Just to be consistent across SDKs I think we should make the default whatever the equivalent for undefined in python